user/views.py

```diff
@@ -43,12 +43,35 @@ def post(self, request):
             if User.objects.filter(account = data['account']).exists():
                 user = User.objects.get(account = data['account'])
                 if bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
-                    access_token = jwt.encode({'ID' : user.id}, SECRET['secret'], ALGORITHM['algorithm']).decode('utf-8')
+                    access_token = jwt.encode({'ID' : user.id}, SECRET_KEY['secret'], ALGORITHM['algorithm']).decode('utf-8')
                     return JsonResponse({"TOKEN": access_token}, status = 200)
```

Comment:
그냥 token 보다는 access_token이 더 명확한 표현입니다. 나중에 refreash token도 같이 사용할 경우, "token" 만으로는 어떤 token이 access token인지 알기 어렵습니다!
```suggestion
                    return JsonResponse({"access_token": access_token}, status = 200)
```